src/services/spotify/models.ts

```diff
@@ -62,4 +62,9 @@ interface PlaylistItem {
     track: Track
 }
 
-export type { User, Playlist, Track, TrackFeatures, PlaylistItem }
+interface BPM {
```

Comment:
This file was meant for spotify api models. This would be better in bpminput or cache